scripts/qa_crowbarsetup.sh

```diff
@@ -2595,7 +2595,7 @@ function custom_configuration
             if [[ $hacloud = 1 ]] ; then
                 # fetch one of the compute nodes as cinder_volume
                 local cinder_volume=("${unclustered_nodes[@]}")
-                if [[ "$deployceph" ]] && iscloudver 7plus ; then
+                if [[ "$deployceph" || $cinder_backend = "nfs" ]] && iscloudver 7plus ; then
```

Comment:
technically I would prefer the $cinder_backend in quotes, but I know its not necessary with [[]] - but shouldn't this be -o instead of '||' ?

Reply:
`||` works in `[[ ]]`, unlike `-o` actually